daemon/cluster/executor/container/container.go

```diff
@@ -75,6 +90,14 @@ func (c *containerConfig) spec() *api.ContainerSpec {
 	return c.task.Spec.GetContainer()
 }
 
+func (c *containerConfig) nameOrID() string {
```

Comment:
We don't know the id in case of container executor but in case of attachment executor we get the containerID as part of `AttachNetwork` resourceapi and the `TaskSpec` carries it.